packages/mermaid/src/docs/.vitepress/scripts/fetch-contributors.ts

```diff
@@ -1,6 +1,8 @@
 // Adapted from https://github.dev/vitest-dev/vitest/blob/991ff33ab717caee85ef6cbe1c16dc514186b4cc/scripts/update-contributors.ts#L6
 
 import { writeFile } from 'node:fs/promises';
+import { knut, plainTeamMembers } from '../data.js';
```

Comment:
I would call this file `teamMembers` rather than `data`. Minor, apart from this everything is ok

Reply:
Done!